components/WeatherIcon.js

```diff
@@ -0,0 +1,266 @@
+    />
+  ),
+  61: (
+    <Player
+      autoplay
+      loop
+      src={rainAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  63: (
+    <Player
+      autoplay
+      loop
+      src={rainAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  65: (
+    <Player
+      autoplay
+      loop
+      src={heavyRainAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  66: (
+    <Player
+      autoplay
+      loop
+      src={rainAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  67: (
+    <Player
+      autoplay
+      loop
+      src={heavyRainAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  71: (
+    <Player
+      autoplay
+      loop
+      src={ligthSnowAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  73: (
+    <Player
+      autoplay
+      loop
+      src={snowAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  75: (
+    <Player
+      autoplay
+      loop
+      src={snowAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  77: (
+    <Player
+      autoplay
+      loop
+      src={snowAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  80: (
+    <Player
+      autoplay
+      loop
+      src={rainAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  81: (
+    <Player
+      autoplay
+      loop
+      src={rainAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  82: (
+    <Player
+      autoplay
+      loop
+      src={heavyRainAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  85: (
+    <Player
+      autoplay
+      loop
+      src={ligthSnowAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  86: (
+    <Player
+      autoplay
+      loop
+      src={snowAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  95: (
+    <Player
+      autoplay
+      loop
+      src={thunderstormAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  96: (
+    <Player
+      autoplay
+      loop
+      src={thunderAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+  99: (
+    <Player
+      autoplay
+      loop
+      src={thunderAnimation}
+      aria-hidden="true"
+      style={{ height: "100px", width: "100px" }}
+    />
+  ),
+};
+
+function WeatherIcon({ weatherData }) {
```

Comment:
please add the export here, keep your code consistent (use the same writing patterns everywhere)
```suggestion
export default function WeatherIcon({ weatherData }) {
```